src/components/publishing/embed.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+
+interface EmbedProps {
+  section: any
+}
+
+const Embed: React.SFC<EmbedProps> = props => {
+  const { url, height, mobile_height } = props.section
+  return <IFrame src={url} scrolling="no" frameBorder="0" height={height} mobileHeight={mobile_height} />
+}
+
+interface FrameProps extends React.HTMLProps<HTMLIFrameElement> {
+  mobileHeight?: number
+  height: number
+}
+
+const iframe: StyledFunction<FrameProps & React.HTMLProps<HTMLIFrameElement>> = styled.iframe
```

Comment:
You’re adding the `React.HTMLProps<HTMLIFrameElement>` props twice, once here and once on line 13.